warehouse/upload.go

```diff
@@ -564,7 +564,10 @@ func (job *UploadJob) run() (err error) {
 			})
 
 			rruntime.GoForWarehouse(func() {
-				specialTables := append(userTables, identityTables...)
+				var specialTables []string
```

Comment:
We could preallocate here specifying the desired `capacity`, right? Basically by using the 3rd argument of [make](https://go.dev/tour/moretypes/13) like [this](https://go.dev/play/p/8MV8RK5g2Wj).